samples/dashboards/demo/accounting/demo.html

```diff
@@ -0,0 +1,15 @@
+<script src="https://code.highcharts.com/dashboards/dashboards.js"></script>
+<script src="https://code.highcharts.com/dashboards/modules/math-modifier.js"></script>
+<script src="https://code.highcharts.com/dashboards/datagrid.js"></script>
+<script src="https://code.highcharts.com/stock/highstock.js"></script>
+<script src="https://code.highcharts.com/highcharts-more.js"></script>
+<script src="https://code.highcharts.com/modules/accessibility.js"></script>
+<script src="https://code.highcharts.com/dashboards/modules/dashboards-plugin.js"></script>
+
+<div id="container"></div>
+<div id="popup">
```

Comment:
In the dashboards you have the build-in popup, was it not useful in this demo?

Reply:
If you mean the possibility of creating a popup based on the abstract `BaseForm` class, then it is possible, but in my opinion it is style over substance. I think my idea needs much less code that may cause confusion for users.

Reply:
Being honest, I'm not convinced, however if its simpler, we can leave like it is.